apps/synapse-portal-framework/src/pages/FileEntityPage/FileEntityPage.tsx

```diff
@@ -0,0 +1,86 @@
+import { useGetPortalComponentSearchParams } from '@/utils/UseGetPortalComponentSearchParams'
+import { Box, Container, Stack } from '@mui/material'
+import { ProvenanceGraph } from 'synapse-react-client'
+import MarkdownSynapse from 'synapse-react-client/components/Markdown/MarkdownSynapse'
+import { useGetEntityBundle } from 'synapse-react-client/synapse-queries'
+import CollapsibleSection from '@/components/CollapsibleSection'
+import { DetailsPageSectionLayoutType } from '@/components/DetailsPage/DetailsPageSectionLayout'
+import { useLocation, useNavigate } from 'react-router'
+import { SynapseSpinner } from 'synapse-react-client/components/LoadingScreen/LoadingScreen'
+import EntityPageMenu from '../../components/EntityPageMenu'
+
+function FileEntityPage() {
+  const searchParams = useGetPortalComponentSearchParams()
+  const entityId = searchParams?.entityId
+  const version = searchParams?.version
+    ? Number(searchParams.version)
+    : undefined
+  const location = useLocation()
+  const navigate = useNavigate()
+
+  const { data: entityBundle, isLoading } = useGetEntityBundle(
+    entityId,
+    version,
+    {
+      includeEntity: true,
+      includeRootWikiId: true,
+    },
+  )
+
+  const fileEntityPageSections = [
+    entityBundle?.rootWikiId && { id: 'wiki', title: 'Wiki' },
+    { id: 'provenance', title: 'Provenance' },
+  ].filter(Boolean) as DetailsPageSectionLayoutType[]
+
+  if (isLoading) {
+    return (
+      <Stack sx={{ my: 3 }}>
+        <SynapseSpinner size={40} />
+      </Stack>
+    )
+  }
+  return (
+    <Container>
+      <Box
+        sx={{
+          display: 'flex',
+          gap: '50px',
+          padding: '50px 0px 80px 0px',
+        }}
```

Comment:
Seems like the style for this layout (which needs to be the same as the details page so the menu layout is correct) should come from a classname or a shared styled component.